tools/python/post_generation/requirements.txt

```diff
@@ -0,0 +1 @@
+-r ../mwm/requirements.txt
```

Comment:
абсолютный путь здесь нельзя указать?
через OMIM_ROOT, например

Reply:
не знаю, погуглил ответа не нашел, может и нельзя
OMIM_ROOT - переменная CMake, это разные подсистемы

Reply:
Я имел в виду сделать OMIM_ROOT для requirements, если здесь начинают появляться такие зависимости.

Reply:
https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format
Since version 10, pip supports the use of environment variables inside the requirements file.

Reply:
up to you

Reply:
я понял, вроде как нельзя

Reply:
максимум только так будет, все requirements.txt будут на одной глубине